outbound/hmail.js

```diff
@@ -43,6 +43,8 @@ const obc = require('./config');
 
 function dummy_func () {}
 
+const CRLF = '\r\n';
```

Comment:
same here, unhoist this.

Reply:
@msimerson I don't know if Sonar was being smart enough, but I could see potential use for that in the rest of the program as a global constant. CRLF will always be CRLF; unless there's some weird thing with CR/LF Mac-Unix-Windows fight going on.

Reply:
If that EVER happens, the programmer using it elsewhere will hoist the variable. Or perhaps they'll do one better and move it over to `haraka-constants`.

Reply:
Moved back for now, as requested.